build.gradle

```diff
@@ -572,6 +605,23 @@ subprojects {
             dependency "software.amazon.awssdk:s3:2.20.29"
             dependency "com.github.rholder:guava-retrying:2.0.0"
 
+            dependency "org.apache.rocketmq:rocketmq-acl:$rocketmq_version"
+            dependency("org.apache.rocketmq:rocketmq-broker:$rocketmq_version") {
+                exclude group: 'ch.qos.logback', name: 'logback-classic'
+            }
+            dependency "org.apache.rocketmq:rocketmq-client:$rocketmq_version"
+            dependency "org.apache.rocketmq:rocketmq-common:$rocketmq_version"
+            dependency "org.apache.rocketmq:rocketmq-filter:$rocketmq_version"
+            dependency "org.apache.rocketmq:rocketmq-logging:$rocketmq_version"
+            dependency("org.apache.rocketmq:rocketmq-namesrv:$rocketmq_version") {
+                exclude group: 'ch.qos.logback', name: 'logback-classic'
+            }
+            dependency "org.apache.rocketmq:rocketmq-remoting:$rocketmq_version"
+            dependency "org.apache.rocketmq:rocketmq-srvutil:$rocketmq_version"
+            dependency "org.apache.rocketmq:rocketmq-store:$rocketmq_version"
+            dependency("org.apache.rocketmq:rocketmq-tools:$rocketmq_version") {
+                exclude group: 'ch.qos.logback', name: 'logback-classic'
+            }
         }
```

Comment:
You've added so much deps from RocketMQ here🤣. Most of them are only used in `eventmesh-storage-plugin/eventmesh-storage-rocketmq/build.gradle`. Can we manage these deps excluding there?

Reply:
Multiple subprojects use RocketMQ:

- `eventmesh-connectors:eventmesh-connector-rocketmq`
- `eventmesh-retry:eventmesh-retry-rocketmq`
- `eventmesh-storage-plugin:eventmesh-storage-rocketmq`

so I moved the exclusion to the dependency management.

AFAIK RocketMQ does not have a `rocketmq-bom` artifact like ([`junit-bom`](https://mvnrepository.com/artifact/org.junit/junit-bom/5.10.1) or [`spring-framework-bom`](https://mvnrepository.com/artifact/org.springframework/spring-framework-bom/6.1.2)) that could be used in the `imports` section above. Adding [`rocketmq-all`](https://mvnrepository.com/artifact/org.apache.rocketmq/rocketmq-all/5.1.4) to the imports would set the version for other unrelated dependencies.

Reply:
I think managing RocketMQ deps in these 3 modules will be better. RocketMQ is one of the connectors/storage-plugins supported by EventMesh, but EventMesh Runtime is not implemented by RocketMQ. 

For example, EventMesh supports Pulsar or Kafka too, however you won't see their deps in `eventmesh/build.gradle`.

Reply:
Ok, I'll move those deps back to the submodules.

This is mostly a question of taste, since (at least in Maven) dependency management is not inherited by consumers of the artifacts (only by child POMs).

BTW: I submitted apache/rocketmq#7732 to add a `rocketmq-bom` artifact, which will allow to specify the version of all RocketMQ artifacts at once.